src/test/java/com/google/gcp_variant_transforms/library/HeaderIteratorTest.java

```diff
@@ -0,0 +1,122 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.library;
+
+import static com.google.common.truth.Truth.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.google.common.collect.ImmutableList;
+import com.google.gcp_variant_transforms.library.HeaderIterator;
+import org.junit.*;
+
+/**
+ * Units tests for HeaderIterator.java
+ */
+public class HeaderIteratorTest {
+
+  public HeaderIterator createHeaderIterator(int numHeaderLines){
+    ImmutableList.Builder<String> headerLinesBuilder = new ImmutableList.Builder<String>();
+    HeaderIterator headerIterator;
+    for (int i = 0; i < numHeaderLines; i++) {
+      headerLinesBuilder.add("##sampleHeaderLine=" + i);
+    }
+    headerIterator = new HeaderIterator(headerLinesBuilder.build());
+    return headerIterator;
```

Comment:
remove HeaderIterator creation and just return it here. ie.

return new HeaderIterator(headerLinesBuilder.build());

Reply:
resolved